app/components/maps/SidebarReplays.tsx

```diff
@@ -33,6 +33,12 @@ export const SidebarReplays = ({
     const [visible, setVisible] = useState(false);
     const [visibleReplays, setVisibleReplays] = useState<FileResponse[]>([]);
 
+    useEffect(() => {
+        // initialize visible replays with the first 25
+        const initiallyVisibleReplays = replays.slice(0, 25);
```

Comment:
The `25` should perhaps be moved to a constant as it is also used in line 206 for the page size